lang/go/go.talon

```diff
@@ -171,7 +243,10 @@ append <user.text> [over]:
 state (air | err): "err"
 error: " err "
 loop over [<user.text>] [over]:
-    insert("forr ")
+    insert("forr")
+    sleep(100ms)
+    key(tab)
+    sleep(100ms)
```

Comment:
What is this doing?

Reply:
This one uses a go-specific vscode snippet, tabs through the positions, and inserts the name at the right slot.
I wondered if this should go in a separate .talon file, triggered on the combination of golang AND vscode,
to allow for snippets from different editors.